docs/index.md

```diff
@@ -70,6 +70,26 @@ metrics:
   `true`.
 - `connect_to_reserved_ips`: Allow reserved IP addresses for requests to
   sources. See [Security](#security). Defaults to `false`.
+- `caches`: Fine-tune cache expiry.
+   - `downloaded`: Fine-tune caches for downloaded files.
+      - `max_unused_for`: Maximum duration to keep a file since last
```

Comment:
It is unclear to me how we can actually set these timeouts.  In the serde structs they are `Option<std::time::Duration>`.  This may need fixing.

Reply:
The way we do it in Relay is use plain numbers in the config:

https://github.com/getsentry/relay/blob/475e6232f474bfe5d18b894b9ccd87c48fd808a1/relay-config/src/config.rs#L462-L464

And then convert it in a getter function:

https://github.com/getsentry/relay/blob/475e6232f474bfe5d18b894b9ccd87c48fd808a1/relay-config/src/config.rs#L1235-L1237

Reply:
As that affects existing code I'll do this in a separate pull request and just leave this as-is here as this is no regression.